src/Native/Runtime/unix/UnixContext.cpp

```diff
@@ -207,12 +207,12 @@ static void RegDisplayToUnwindContext(REGDISPLAY* regDisplay, unw_context_t *unw
     // initialized by unw_init_local(), which are not updated by
     // unw_set_reg().
 
-#define ASSIGN_REG(regIndex, regName) \
-    unwContext->regs[regIndex] = regDisplay->regName2;
+#define ASSIGN_REG(regIndex, regNamePre) \
```

Comment:
I don't understand why we pass in register names "S" and "I" and then add the "P" here. The register names we use are SP and IP, so why would we do this?